styles/globals.css

```diff
@@ -0,0 +1,36 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
+@layer base {
+  /* Font families */
+  h1, h2, h3, h4, h5 {
+    @apply font-display font-bold;
```

Comment:
Minor change here but [the Canada.ca typography page](https://design.canada.ca/styles/typography.html) is technically contradicting itself. 

It says to use `bold`, but 
- `font-weight: bold` === `font-weight: 700` ([MDN link](https://developer.mozilla.org/en-US/docs/Web/CSS/font-weight))

If you look at the CSS for that typography page, you can see that they are actually using `font-weight: 600`. So we should use that instead of "bold". 

If there is a "report a problem" widget on that typography page, we should do it ;)

Reply:
Changed from font-bold (700) to font-semibold